### ISSUE TITLE
Temperature must be between 5 and 90

### ISSUE THREAD

**Issue by Ucsus** [make-all/tuya-local#1198]
Avatto thermostat, protocol 3.3, ip auto
When I try to change the temperature, I get an error "Failed to call service climate/set_temperature.temperature (0.0) must be between 5 and 90" and the temperature does not change until I move the upper temperature slider. Temperature limits range from 5 to 30.

```
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/commands.py:226
Integration: Home Assistant WebSocket API ([documentation](https://www.home-assistant.io/integrations/websocket_api), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+websocket_api%22))
First occurred: 00:24:45 (4 occurrences)
Last logged: 09:36:51

[140718256828992] temperature (0.0) must be between 5 and 90
[140718662361152] temperature (0.0) must be between 5 and 90
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 226, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 2012, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2049, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 235, in handle_service
    return await service.entity_service_call(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 876, in entity_service_call
    response_data = await _handle_entity_call(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 948, in _handle_entity_call
    result = await task
             ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 632, in async_service_temperature_set
    await entity.async_set_temperature(**kwargs)
  File "/config/custom_components/tuya_local/climate.py", line 220, in async_set_temperature
    await self.async_set_target_temperature(
  File "/config/custom_components/tuya_local/climate.py", line 232, in async_set_target_temperature
    await self._temperature_dps.async_set_value(
  File "/config/custom_components/tuya_local/helpers/device_config.py", line 464, in async_set_value
    settings = self.get_values_to_set(device, value)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tuya_local/helpers/device_config.py", line 933, in get_values_to_set
    raise ValueError(f"{self.name} ({value}) must be between {mn} and {mx}")
ValueError: temperature (0.0) must be between 5 and 90
```
![error](https://github.com/make-all/tuya-local/assets/7962244/4a8e798a-e6f1-40cf-8f14-40e804c786d4)

**Comment by make-all** [comment]
It is not clear where the error is coming from.  From the look of the screenshot, this is a custom card, is the problem reproducible with the standard climate card?

If it is, please download the diagnostics for the device.

**Comment by KTibow** [comment]
> It is not clear where the error is coming from. From the look of the screenshot, this is a custom card

That screenshot is from the vanilla more info dialog.

**Comment by make-all** [comment]
No diagnostics were received, so there is nothing more that can be done, though this may be fixed by the change in 0bb592db that was made for the optional max/min temperature dps in the Avatto ME81 thermometer
